src/accelerate/accelerator.py

```diff
@@ -1918,3 +1918,13 @@ def optimizer_step_was_skipped(self):
             if optimizer.step_was_skipped:
                 return True
         return False
+
+    def skip_batches(self, dataloader, num_batches: int = 0):
+        """
+        Creates a new `torch.utils.data.DataLoader` that will efficiently skip the first `num_batches`.
+
+        Args:
+            dataloader (`torch.utils.data.DataLoader`): The data loader in which to skip batches.
+            num_batches (`int`, *optional*, defaults to 0): The number of batches to skip
```

Comment:
Docstring format nit
```suggestion
            dataloader (`torch.utils.data.DataLoader`): 
                The data loader in which to skip batches.
            num_batches (`int`, *optional*, defaults to 0): 
                The number of batches to skip
```